pkg/ocm/client_test.go

```diff
@@ -142,15 +154,15 @@ var _ = Describe("OCM Client", func() {
 	})
 
 	Context("When getting upgrade policies", func() {
-		It("returns the correct info", func() {
+		It("retrieves OSD type only", func() {
 
 			upResponder, _ := httpmock.NewJsonResponder(http.StatusOK, upgradePolicyListResponse)
 			upUrl := path.Join(CLUSTERS_V1_PATH, TEST_CLUSTER_ID, UPGRADEPOLICIES_V1_PATH)
 			httpmock.RegisterResponder(http.MethodGet, upUrl, upResponder)
 
 			result, err := oc.GetClusterUpgradePolicies(TEST_CLUSTER_ID)
 
-			Expect(*result).To(Equal(upgradePolicyListResponse))
+			Expect(result.Items[0]).To(Equal(upgradePolicyListResponse.Items[0]))
```

Comment:
This isn't necessarily verifying that only the OSD policy was returned, just that the OSD policy was the first one in the list. Perhaps it could also verify that the length of the list returned is 1.